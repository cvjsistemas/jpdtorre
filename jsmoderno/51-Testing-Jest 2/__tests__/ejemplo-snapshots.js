const cliente ={
    nombre:'Juan P',
    balance:500,
    tipo:'premium'
}

describe('testing al cliente',()=>{
    test('comprobar objeto',()=>{
        expect(cliente).toMatchSnapshot();
    });
})