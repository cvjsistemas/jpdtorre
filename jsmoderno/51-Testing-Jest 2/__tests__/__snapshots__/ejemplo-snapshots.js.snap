// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing al cliente comprobar objeto 1`] = `
{
  "balance": 500,
  "nombre": "Juan P",
  "tipo": "premium",
}
`;
